highfidelity/hifi
# Case 19473: Reduce branching in our shaders

https://highfidelity.manuscript.com/f/cases/19473/Reduce-branching-in-our-shaders

Test plan:
- Confirm that the following work as in master:
     - luci.js option toggles
     - ambient occlusion (controlled from luci)
     - VR rendering
     - Avatar fading
     - Hand lasers and teleport parabola rendering
     - Text entities rendering
     - Skybox rendering (with color, texture, shadows, and haze)
- Run the particle tests: https://github.com/highfidelity/hifi_tests/tree/master/tests/content/entity/particleEffect
- Run the rendering tests: https://github.com/highfidelity/hifi_tests/tree/master/tests/engine/render
